addon/components/visualizer-course-objectives.js

```diff
@@ -28,6 +34,28 @@ export default class VisualizerCourseObjectives extends Component {
     return this.courseSessions.slice();
   }
 
+  get tableData() {
+    if (!this.dataObjects) {
+      return [];
+    }
+    return this.dataObjects.map((obj) => {
+      const rhett = {};
+      rhett.minutes = obj.data;
+      // KLUDGE!
+      // multiply by 1,000 to get everything back to full numbers.
+      // that way, we can rely on string sorting rather than having to implement our own
+      // sorting callback.
+      // [ST 2022/10/06]
+      rhett.percentage = obj.percentage * 1000;
+      rhett.percentageLabel = obj.label;
+      rhett.objective = obj.meta.courseObjective.title;
+      rhett.competency = obj.meta.competency?.title;
+      rhett.sessionTitles = mapBy(obj.meta.sessionObjectives, 'sessionTitle').sort().join(', ');
```

Comment:
This should be either lcoaleSort or using our sortBy helper so sessions with a leading number get sorted as expected.

Reply:
:heavy_check_mark: